src/main/java/com/theironyard/war/Player.java

```diff
@@ -0,0 +1,42 @@
+package com.theironyard.war;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by EddyJ on 6/23/16.
+ */
+public class Player {
+    String name;
```

Comment:
These need to be private.  Do not allow other classes to access the player's attributes directly.